src/coreclr/inc/jithelpers.h

```diff
@@ -267,7 +267,7 @@
     JITHELPER(CORINFO_HELP_EE_PRESTUB,          ThePreStub,                 CORINFO_HELP_SIG_NO_ALIGN_STUB)
 
 #if defined(HAS_FIXUP_PRECODE)
-    JITHELPER(CORINFO_HELP_EE_PRECODE_FIXUP,    PrecodeFixupThunk,          CORINFO_HELP_SIG_NO_ALIGN_STUB)
+    JITHELPER(CORINFO_HELP_EE_PRECODE_FIXUP,    ThePreStub,                 CORINFO_HELP_SIG_NO_ALIGN_STUB)
```

Comment:
All these `CORINFO_HELP_EE_...` entries can be deleted. They are left-over from fragile NGen.